src/_internals/function-serializer.ts

```diff
@@ -1,14 +1,13 @@
-import { ShopwareMessageTypes } from '../messages.types';
-import { send, ShopwareMessageSendData, handleFactory } from '../channel';
+import { send, handleFactory } from '../channel';
 import { traverseObject, isObject, generateUniqueId } from './utils';
 
 /* eslint-disable */
-export function serializeMessageData<MESSAGE_TYPE extends keyof ShopwareMessageTypes>(messageData: ShopwareMessageSendData<MESSAGE_TYPE>): void {
+export function serializeFunctionsInMessageData<MESSAGE_DATA extends object>(messageData: MESSAGE_DATA): void {
```

Comment:
I solved the problem this way:
Serializers export a default Object with 2 functions: serialize / deserialize in the channel this looks like this:

EntitySerializer.serialize(messageData);
FunctionSerializer.serialize(messageData);

I found this quiet charming maybe we want to consider this?